src/mds/nameserver2/curvefs.cpp

```diff
@@ -566,7 +574,7 @@ StatusCode CurveFS::IsSnapshotAllowed(const std::string &fileName) {
 
     // the client version satisfies the conditions
     std::string clientVersion;
-    bool exist = fileRecordManager_->GetFileClientVersion(
+    bool exist = fileRecordManager_->GetMinimumFileClientVersion(
```

Comment:
Can it be guaranteed that min is a write node?

Reply:
nope